### ISSUE TITLE
With 2.2.4 when battery reach 100% I'm flooded by messages 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#185]
```
This is what I had, I removed the notification so I'm not flooded any more

GTsms
Batterie à 100%
Nouveau message de disponibilité de GTsms - GTalkSMS - 100% - USB   10:11

GTsms
Batterie à 100%
Batterie à 100%
Batterie à 100%
Batterie à 100%
10:11

GTsms
Batterie à 100%
Batterie à 100%
Batterie à 100%
Batterie à 100%
Batterie à 100%
10:12

GTsms
Batterie à 100%
Batterie à 100%
Batterie à 100%
Batterie à 100%
```

Original issue reported on code.google.com by `lpenou.tstXX@mpo.li` on 15 Jun 2011 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
Issue 195 has been merged into this issue.
```

Original comment by `fschm...@gmail.com` on 29 Jun 2011 at 4:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 11 Jul 2011 at 4:26
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.0
```

Original comment by `Florent....@gmail.com` on 15 Jul 2011 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
